genSizing.js

```diff
@@ -7,6 +7,7 @@ const SIZING_FILENAME = "sizing.less";
 const SPACING_FILENAME = "spacing.less";
 const sizes = {
   none: 0,
+  "4xs": 0.0625,
```

Comment:
had to add this, since the auto-generated file had been manually edited without updating here